### ISSUE TITLE
CircuitBreakerMetrics.getNumberOfSlowFailedCalls returns wrong negative values.

### ISSUE THREAD

**Issue by jaemok** [resilience4j/resilience4j#1335]
Resilience4j version: 1.6.1

Java version: openjdk 11

When I monitor "resilience4j_circuitbreaker_slow_calls", it is fine.
When I monitor "resilience4j_circuitbreaker_slow_calls[kind=\"failed\"]"  or "resilience4j_circuitbreaker_slow_calls[kind=\"successful\"]", they show quadratic decrease and increase.

getNumberOfSlowSuccessfulCalls returns positive numbers and it increases quadratically.
getNumberOfSlowFailedCalls returns negative numbers and it decreases quadratically.

And the reason is, resilience4j core did not reset the bucket when FixedSizeSlidingWindowMetrics.moveWindowByOne is called.

https://github.com/resilience4j/resilience4j/blob/49219fca21cd02b05cf3fc80ad9b9690ccd96009/resilience4j-core/src/main/java/io/github/resilience4j/core/metrics/Measurement.java#L23-L28

it should be updated to
```
 void reset() { 
     this.totalDurationInMillis = 0; 
     this.numberOfSlowCalls = 0; 
     this.numberOfFailedCalls = 0;
     this.numberOfSlowFailedCalls = 0;   // check this line
     this.numberOfCalls = 0; 
 } 
```

**Comment by jaemok** [comment]
This is same issue here. https://github.com/resilience4j/resilience4j/issues/1262
https://github.com/resilience4j/resilience4j/commit/e95c4773d1bf23d2995792a8db18770f071781c3 fixed SlidingTimeWindowMetrics.
But the problem remains for FixedSizeSlidingWindowMetrics which uses Measurement instead of PartialAggregation.

**Comment by dlsrb6342** [comment]
Hi @jaemok 

Right. Thank you for reporting. Would you like to raise a PR?

**Comment by dlsrb6342** [comment]
Fixed by #1342
Thanks for your contribution.
